api/openapi-spec/swagger.json

```diff
@@ -1996,7 +1996,7 @@
       "type": "object"
     },
     "io.k8s.api.authentication.v1alpha1.SelfSubjectReview": {
```

Comment:
hmm... the beta type should have shown up in this file

Reply:
After adding API to the disabled by default list everything is ok now  https://github.com/kubernetes/kubernetes/pull/116274#discussion_r1128231647